<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>711</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_choose"/>
   <addaction name="action_clear"/>
   <addaction name="action_pre"/>
   <addaction name="action_next"/>
   <addaction name="action_left"/>
   <addaction name="action_right"/>
   <addaction name="action_big"/>
   <addaction name="action_small"/>
   <addaction name="action_all"/>
   <addaction name="action_return"/>
   <addaction name="action_play"/>
   <addaction name="action_edit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_choose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/1.png</normaloff>:/icons/1.png</iconset>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
   <property name="toolTip">
    <string>选择路径</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/2.png</normaloff>:/icons/2.png</iconset>
   </property>
   <property name="text">
    <string>清除路径</string>
   </property>
   <property name="toolTip">
    <string>清除路径</string>
   </property>
  </action>
  <action name="action_pre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/3.png</normaloff>:/icons/3.png</iconset>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
   <property name="toolTip">
    <string>上一张</string>
   </property>
  </action>
  <action name="action_next">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/4.png</normaloff>:/icons/4.png</iconset>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
   <property name="toolTip">
    <string>下一张</string>
   </property>
  </action>
  <action name="action_left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/5.png</normaloff>:/icons/5.png</iconset>
   </property>
   <property name="text">
    <string>左旋</string>
   </property>
   <property name="toolTip">
    <string>左旋</string>
   </property>
  </action>
  <action name="action_right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/6.png</normaloff>:/icons/6.png</iconset>
   </property>
   <property name="text">
    <string>右旋</string>
   </property>
   <property name="toolTip">
    <string>右旋</string>
   </property>
  </action>
  <action name="action_big">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/7.png</normaloff>:/icons/7.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="action_small">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/8.png</normaloff>:/icons/8.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/9.png</normaloff>:/icons/9.png</iconset>
   </property>
   <property name="text">
    <string>全屏模式</string>
   </property>
   <property name="toolTip">
    <string>全屏模式</string>
   </property>
  </action>
  <action name="action_return">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/10.png</normaloff>:/icons/10.png</iconset>
   </property>
   <property name="text">
    <string>返回原始大小</string>
   </property>
   <property name="toolTip">
    <string>返回原始大小</string>
   </property>
  </action>
  <action name="action_play">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/11.png</normaloff>:/icons/11.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="toolTip">
    <string>播放</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/12.png</normaloff>:/icons/12.png</iconset>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
   <property name="toolTip">
    <string>edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
